pandas/core/generic.py

```diff
@@ -6527,9 +6527,11 @@ def replace(
             0.
         inplace : bool, default False
             Update the data in place if possible.
-        limit_direction : {'forward', 'backward', 'both'}, default 'forward'
+        limit_direction : {'forward', 'backward', 'both'}, default is None
```

Comment:
Should revert this; though it shows `None` as the value it still does default to `forward` no?

Reply:
Thanks for spotting this. Will be fixed in next series of commits.